tests/php/unit/Dashbaord/CalendarWidgetTest.php

```diff
@@ -23,33 +23,69 @@
  */
 namespace OCA\Calendar\Dashboard;
 
-use ChristophWurst\Nextcloud\Testing\TestCase;
 use OCA\Calendar\Service\JSDataService;
-use OCP\IInitialStateService;
+use OCA\DAV\CalDAV\CalendarImpl;
+use OCP\AppFramework\Services\IInitialState;
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\Calendar\IManager;
+use OCP\Dashboard\IButtonWidget;
+use OCP\Dashboard\Model\WidgetItem;
+use OCP\IDateTimeFormatter;
 use OCP\IL10N;
+use OCP\IURLGenerator;
+use PHPUnit\Framework\MockObject\MockObject;
+use Safe\DateTimeImmutable;
+use Test\TestCase;
 
 class CalendarWidgetTest extends TestCase {
 
-	/** @var IL10N|\PHPUnit\Framework\MockObject\MockObject */
+	/** @var IL10N|IL10N&MockObject|MockObject  */
```

Comment:
what is this `&` notation?

Reply:
done automatically by my IDE, maybe a setting.